docs/research/414-prompt-examples/score.py

```
@@ -0,0 +1,175 @@
+def parse(path):
+    """(fixture, run, raw, output, outcome) for every call in a `show` capture."""
+    out, fx, raw, cur = [], None, None, None
+    def flush():
+        nonlocal cur
+        if cur:
+            out.append(dict(fixture=fx, run=cur[0], output="\n".join(cur[1]).strip(),
+                            outcome=cur[2], raw=raw))
+            cur = None
+    for line in Path(path).read_text(encoding="utf-8").split("\n"):
+        m = re.match(r"^━━ \[(.+?)\]", line)
+        if m: flush(); fx = m.group(1); continue
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)